src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function convertSalary(salary) {
+  return +salary.slice(1).split(',').join('');
+};
+
+const table = document.querySelector('table');
+const tableBody = document.querySelector('tbody');
+const rows = [...tableBody.querySelectorAll('tr')];
+
+table.addEventListener('click', (e) => {
+  const header = e.target.closest('th');
+
+  if (header) {
+    switch (e.target.textContent) {
+      case 'Name':
+        rows.sort((a, b) =>
+          a.children[0].textContent
+            .localeCompare(b.children[0].textContent));
+        break;
+
+      case 'Position':
+        rows.sort((a, b) =>
+          a.children[1].textContent
+            .localeCompare(b.children[1].textContent));
+        break;
```

Comment:
Look at the duplicate code you have here. Move everything that is duplicated to the helper function